src/nvim/indent_c.c

```diff
@@ -508,29 +505,25 @@ get_indent_nolabel (     /* XXX */
  *   label:	if (asdf && asdfasdf)
  *		^
  */
-static int skip_label(linenr_T lnum, char_u **pp)
-{
-  char_u      *l;
-  int amount;
-  pos_T cursor_save;
+static int get_indent_after_label_pp(linenr_T lnum, char_u **pp)
+  FUNC_ATTR_NONNULL_ALL
+{ /* XXX */
+  char_u *l = ml_get(lnum);
+  int indent;
 
-  cursor_save = curwin->w_cursor;
-  curwin->w_cursor.lnum = lnum;
-  l = get_cursor_line_ptr();
-  /* XXX */
+  // We repeatedly call "ml_get(lnum)" because every time it's called (or any
+  // time we call a function that calls it, marked with an 'XXX' comment) the
+  // returned pointer is invalidated.
```

Comment:
I didn't find any tabs here to change to spaces, can you tell me which tabs you mean?
Weirdly, the convention throughout the file is to add the `/* XXX */` comment whenever you might invalid an `ml_get_*()` pointer. It took me forever to realize this though. I've added one of those comments to this function's definition and also inline with the `cin_islabel()` call (which is the thing on line 517 that calls ml_get). How's it look?